.github/workflows/publish-pacts.yaml

```
@@ -0,0 +1,50 @@
+name: Publish Pacts
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ dev ]
+
+jobs:
+  setup-and-test:
+    runs-on: ubuntu-latest
+    outputs:
+      nb-b64: ${{ steps.encode.outputs.nb-b64 }}
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```

Nit: checkout is up to v3. https://github.com/actions/checkout